pkg/kpt/api/kptfile/v1/types.go

```diff
@@ -210,6 +233,8 @@ type PackageInfo struct {
 	// Relative slash-delimited path to the license file (e.g. LICENSE.txt)
 	LicenseFile string `yaml:"licenseFile,omitempty" json:"licenseFile,omitempty"`
 
+	ReadinessGates []ReadinessGate `yaml:"readinessGates,omitempty" json:"readinessGates,omitempty"`
```

Comment:
This change make kpt packages managed by porch incompatible with kpt packages by kpt, whose Kptfile schema is defined here: https://github.com/kptdev/kpt/blob/main/pkg/api/kptfile/v1/types.go

Reply:
I wasn't able to attend to the latest meetings, do we have a community agreement/decision to break this compatibility?

Reply:
On this, why are we using/defining our own ktp api?
There is similar kpt related stuff in the [nephio repo ](https://github.com/nephio-project/nephio/tree/main/krm-functions/lib/kptfile/v1).
Surely we should try to avoid customising an internal version and look to having a single shared api.